docs/index.d.ts

```diff
@@ -209,6 +209,222 @@ export interface NetworkBlockTimesResponse {
   };
 }
 
+/**
+ * An AccountBalanceRequest is utilized to make a balance request on the /account/balance endpoint. If the block_identifier is populated, a historical balance query should be performed.
+ */
+export interface RosettaAccountBalanceRequest {
+  network_identifier: NetworkIdentifier;
+  account_identifier: RosettaAccount;
+  block_identifier?: RosettaBlockIdentifier;
+}
+
+/**
+ * An AccountBalanceResponse is returned on the /account/balance endpoint. If an account has a balance for each AccountIdentifier describing it (ex: an ERC-20 token balance on a few smart contracts), an account balance request must be made with each AccountIdentifier.
+ */
+export interface RosettaAccountBalanceResponse {
+  block_identifier: RosettaBlockIdentifier;
+  /**
+   * A single account balance may have multiple currencies
+   */
+  balances: RosettaAmount[];
+  /**
+   * If a blockchain is UTXO-based, all unspent Coins owned by an account_identifier should be returned alongside the balance. It is highly recommended to populate this field so that users of the Rosetta API implementation don't need to maintain their own indexer to track their UTXOs.
+   */
+  coins?: RosettaCoin[];
+  /**
+   * Account-based blockchains that utilize a nonce or sequence number should include that number in the metadata. This number could be unique to the identifier or global across the account address.
+   */
+  metadata?: {
+    sequence_number: number;
+    [k: string]: unknown | undefined;
```

Comment:
Are all these `[k: string]: unknown | undefined;` intentional? 
They can be removed by adding `"additionalProperties": false` to the json schemas.